src/main/java/ladder/domain/Result.java

```diff
@@ -0,0 +1,52 @@
+package ladder.domain;
+
+public class Result {
+
+    private final Name name;
+    private final Reward reward;
+
+    public Result(String name, String reward) {
+        this(new Name(name), new Reward(reward));
+    }
+
+    public Result(Name name, Reward reward) {
+        this.name = name;
+        this.reward = reward;
+    }
+
+    public boolean matchName(String name) {
+        return this.name.getValue().equals(name);
```

Comment:
이름이 동일한지는 Name 객체가 책임져야지 Result 객체가 책임질 일은 아닌 것 같습니다.

```suggestion
        return this.name.isEqualsTo(name);
```
or
```suggestion
        return this.name.equals(new Name(name));
```